src/seaport.ts

```diff
@@ -890,20 +914,25 @@ export class Seaport {
     fulfillments,
     overrides,
     accountAddress,
+    domain = "",
   }: {
     orders: (OrderWithCounter | Order)[];
     fulfillments: MatchOrdersFulfillment[];
     overrides?: PayableOverrides;
     accountAddress?: string;
+    domain?: string;
   }): TransactionMethods<
     ContractMethodReturnType<SeaportContract, "matchOrders">
   > {
     const signer = this._getSigner(accountAddress);
 
-    return getTransactionMethods(this.contract.connect(signer), "matchOrders", [
-      orders,
-      fulfillments,
-      overrides,
-    ]);
+    const tag = domain ? getTagFromDomain(domain) : "";
```

Comment:
This ternary feels weird since the default value for domain is an empty string anyways

Reply:
seems fine, the ternary operator is typically used in this way